crowbar_framework/app/models/ceilometer_service.rb

```diff
@@ -32,21 +32,33 @@ def role_constraints
       {
         "ceilometer-agent" => {
           "unique" => false,
-          "count" => -1
+          "count" => -1,
+          "exclude_platform" => {
+            "suse" => "12.0"
```

Comment:
This is not correct: ceilometer-agent should actually be used on SLE12 compute nodes.